tests/ci-csi-cinder-e2e.sh

```diff
@@ -32,7 +32,6 @@ cd "${REPO_ROOT}" || exit 1
 # PULL_NUMBER and PULL_BASE_REF are Prow job environment variables
 PR_NUMBER="${PULL_NUMBER:-}"
 [[ -z $PR_NUMBER ]] && echo "PR_NUMBER is required" && exit 1
-PR_BRANCH="${PULL_BASE_REF:-master}"
```

Comment:
I assume this used to be the branch that related to the submited PR
is it not necessary or you used other variable to replace?

Reply:
um... I didn't check all code but from the naming convention I think we will do something like 
master + PR => new test code thing to be tested on ?

Reply:
Honestly I have no idea, but it does seem this is unused anywhere.
```
mdulko:cloud-provider-openstack/ (dev-environment) $ grep -R "github_pr_branch" .
mdulko:cloud-provider-openstack/ (dev-environment) $ 
```

Reply:
to make it safe, can you help change some code (e.g update a log in the code )
and after the CI run, we check the output and see whether log is really there to prove the PR is running on top of master, if it works then let's remove the log and get this PR merge? 

Thanks

Reply:
@jichenjc: Is there a need? If you take a look at [build-log](https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/cloud-provider-openstack/2472/openstack-cloud-controller-manager-e2e-test/1722951771107627008/build-log.txt) you can see the `ci-occm-e2e.sh` script running with `-x` and there's no `PR_BRANCH` line anymore, yet everything works. It clearly runs the new code.

Reply:
ok, thanks for the tip ,I think that make sense to me